contracts/plugins/trading/DutchTrade.sol

```diff
@@ -45,15 +46,15 @@ contract DutchTrade is ITrade {
 
     TradeStatus public status; // reentrancy protection
 
-    ITrading public origin; // initializer
+    ITrading public origin; // the address that initialized the contract
 
     // === Auction ===
     IERC20Metadata public sell;
     IERC20Metadata public buy;
     uint192 public sellAmount; // {sellTok}
 
     // The auction runs from [startTime, endTime], inclusive
-    uint48 public startTime; // {s} when the dutch auction begins (1 block after init())
+    uint48 public startTime; // {s} when the dutch auction begins (12s after init())
```

Comment:
if we don't yet have a single list of all the places that we consider 1 block == 12s, we should make one

Reply:
it should probably go somewhere in the repo documentation

Reply:
Seems right. I'll create a file in `docs`